monkey/monkey_island/cc/next_ui/server.js

```diff
@@ -6,7 +6,7 @@ const https = require('https');
 const fs = require('fs');
 
 const host = '0.0.0.0';
-const port = process.env.JAVASCRIPT_RUNTIME_PORT;
+const port = process.env.NEXT_PUBLIC_JAVASCRIPT_RUNTIME_PORT;
```

Comment:
Can you import the port from `cc/next_ui/src/app/consts.tsx`?

Reply:
I tried, but it doesn't build then. It was giving me some error.